src/MockClassifier.Api/Interfaces/IClassifierService.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace MockClassifier.Api.Interfaces
+{
+public interface IClassifierService
+{
+    string[] Classify(string messageBody);
+}
+}
```

Comment:
This not an action for @priyankasharma55, but regarding our discussion earlier @decodingahmed, this is the sort of thing that StyleCop would sort out rather than us having to look for it in PRs